packages/fxa-settings/src/components/LinkRememberPassword/index.stories.tsx

```diff
@@ -3,27 +3,28 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import LinkRememberPassword, { LinkRememberPasswordProps } from '.';
+import LinkRememberPassword from '.';
 import { LocationProvider } from '@reach/router';
 import { Meta } from '@storybook/react';
-import { MOCK_EMAIL } from './mocks';
+import { MOCK_ACCOUNT } from '../../models/mocks';
 
 export default {
   title: 'components/LinkRememberPassword',
   component: LinkRememberPassword,
 } as Meta;
 
-const storyWithProps = (props?: LinkRememberPasswordProps) => {
+const storyWithProps = ({ ...props }) => {
   const story = () => (
     <LocationProvider>
-      <LinkRememberPassword {...props} />
+      <LinkRememberPassword
```

Comment:
Ohhhh I see -- `props` doesn't match the `LinkRememberProps` type anymore because some of the values are hardcoded in!